src/styles/_home-page.scss

```diff
@@ -237,5 +241,9 @@
 
   .promoted__cards {
     grid-template-columns: 1fr 1fr 1fr;
+
+    a:hover {
```

Comment:
I realized this change only applied to a particular media query. I fixed this in a different branch.